cli/azd/internal/appdetect/appdetect_test.go

```diff
@@ -173,32 +173,6 @@ func TestDetect(t *testing.T) {
 						JavaProjectOptionWinMavenWrapperPath:   filepath.Join(dir, "java", "mvnw.cmd"),
 					},
 				},
-				{
-					Language:      Java,
```

Comment:
@rujche I found that in the `appdetect_test`, when`createMavenProject` against the java multi-module project, it will be too slow and cost too much time. Could you please help to troubleshoot it, thanks!

![image](https://github.com/user-attachments/assets/32ab4fb2-e555-40bd-8f0f-c7aef48eec06)

![image](https://github.com/user-attachments/assets/f2c8d060-1681-4cfc-99a9-8ab86fbf6744)
https://github.com/azure-javaee/azure-dev/actions/runs/12556279415/job/35008146429#step:5:55

Reply:
Good point.
It will be faster after this PR merged: https://github.com/azure-javaee/azure-dev/pull/99